app/registries/menu/loader.rb

```diff
@@ -96,9 +96,8 @@ def self.load
 
       Manager.map :labs_menu do |menu|
         menu.sub_menu :lab_features_menu, :caption => N_('Lab Features'), :icon => 'fa fa-flask' do
-          menu.item :host_wizard,
-            :caption => 'Host Wizard',
-            :url => '/host_wizard'
+          menu.item :host_wizard, :caption => 'Host Wizard', :url => '/host_wizard'
+          menu.item :architectures, :caption => 'Architectures', :url => '/next_architectures'
```

Comment:
This is just to make the lab page reachable.

Reply:
This was already wrong for `host_wizard, but it's better to reuse the Rails helpers for paths: https://guides.rubyonrails.org/routing.html#generating-paths-and-urls-from-code

Reply:
there is the the `get 'next_architectures' => 'react#index'` in routes.rb which I think is what you refer to.

Reply:
Yes, but you should never use `/next_architectures` (and `/host_wizard` for that matter). That's a bad pattern and helpers should be used.

Reply:
oh right, I guess when we switch the actual page and use a real rails path we could use something like `architectures_path`

Reply:
using `experimental_architectures_path` worked on other files,
in `loader.rb` I guess we need to include some Rails helper?
I also tried doing something like `Rails.application.routes.url_helpers.experimental_architectures_path` which didn't work